### PR TITLE
Fixed #640 - Screen timeout in case of popup mode. 

### LINKED ISSUE #640: Screen turns off when watching a long video in pop-up mode.
It would be nice to leave the screen on in pop-up mode until the video finishes.

### PR DESCRIPTION
**Pull request by krtkush**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

Currently, the screen times out and switches off when a video is playing in pop-up mode. 
With this fix the screen will not time out in that mode. On pausing the video, the time out will work again.

### CODE REVIEW

**Comment by krtkush** [comment]
I'll need some guidance as to why the checks are failing. 

Problem seems to be -

```
Could not resolve all files for configuration ':app:debugCompileClasspath'.
Could not find common.jar (android.arch.core:common:1.1.0).
```

**Comment by theScrabi** [comment]
Yea its Travis, ... it can't get some mvn repos... we have the same issue over at ownCloud atm.

File: app/src/debug/gen/org/schabi/newpipe/BuildConfig.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*___Generated_by_IDEA___*/
```
Comment (theScrabi) [inline app/src/debug/gen/org/schabi/newpipe/BuildConfig.java]: Remove this file

File: app/src/debug/gen/org/schabi/newpipe/Manifest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
```
Comment (theScrabi) [inline app/src/debug/gen/org/schabi/newpipe/Manifest.java]: Remove this file

File: app/src/debug/gen/org/schabi/newpipe/R.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
```
Comment (theScrabi) [inline app/src/debug/gen/org/schabi/newpipe/R.java]: Remove this file

File: app/src/main/gen/org/schabi/newpipe/BuildConfig.java [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/*___Generated_by_IDEA___*/
```
Comment (theScrabi) [inline app/src/main/gen/org/schabi/newpipe/BuildConfig.java]: Remove this file

File: app/src/main/gen/org/schabi/newpipe/Manifest.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
```
Comment (theScrabi) [inline app/src/main/gen/org/schabi/newpipe/Manifest.java]: Remove this file

File: app/src/main/gen/org/schabi/newpipe/R.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
```
Comment (theScrabi) [inline app/src/main/gen/org/schabi/newpipe/R.java]: Remove this file

**Comment by krtkush** [comment]
@theScrabi Made changes as requested.

**Comment by theScrabi** [comment]
bueno gracias :)

**Comment by krtkush** [comment]
@theScrabi Since the branch has been merged, can we consider bug #640 resolved?

**Comment by krtkush** [comment]
Thank you :)

**Pull request merged**